.travis.yml

```diff
@@ -1,40 +0,0 @@
-sudo: required
-services:
-  - docker
-
-before_install:
-  - docker pull gcr.io/cpp-projects/cpp-ci:1
-
-script:
-  - >
-    docker run -v `pwd`:/leatherman gcr.io/cpp-projects/cpp-ci:1 /bin/bash -c "
-    cd /leatherman &&
-    rm locales/leatherman.pot &&
-    cmake $EXTRA_VARS . &&
-    mkdir dest &&
-    make $TARGET DESTDIR=/leatherman/dest -j2 &&
-    { [ '$COVERALLS' == 'ON' ] && coveralls --gcov-options '\-lp' -r . -b . -e src -e vendor >/dev/null || true; }
-    "
-  - if [ "$DO_RELEASE" == "true" ]; then tar czvf leatherman${PKG_SUFFIX}.tar.gz `find dest -type f -print`; fi
-
-env:
-  matrix:
-    - TARGET=cpplint
-    - TARGET=cppcheck
-    - TARGET="all test install ARGS=-v" DO_RELEASE=true PKG_SUFFIX="" EXTRA_VARS="-DBOOST_STATIC=ON"
-    - TARGET="all test install ARGS=-v" DO_RELEASE=true PKG_SUFFIX="-dynamic" EXTRA_VARS="-DLEATHERMAN_SHARED=ON"
-    - TARGET="all test install ARGS=-v" EXTRA_VARS="-DCMAKE_BUILD_TYPE=Debug -DCOVERALLS=ON" COVERALLS=ON
-    - TARGET="all test install ARGS=-v" EXTRA_VARS="-DLEATHERMAN_USE_LOCALES=OFF"
-    - TARGET="all test install ARGS=-v" EXTRA_VARS="-DLEATHERMAN_GETTEXT=OFF"
-
-deploy:
-  provider: releases
-  api_key:
-    secure: k0rU0hNiUu7WqfrqlxP57bOg+phtFEgod/ViE6pJix53+6vpoyOdZzIc2lKCBongysRIAqPsSuNj00/PZxLSkVk2gNxXPpi+AScdh2fheYaWWCmyXuETm+A1nWMUPL7Ep/kIRffIl5I2bmGGqsyidBtdVFeYHp1wZhbHrxjYlYE=
```

Comment:
I assume this api key allows us to push releases to leatherman from travis-ci.com? It looks like the last release was done in https://app.travis-ci.com/github/puppetlabs/leatherman/jobs/577428911#L718 If this key is specific to this repo(?) can we make it so the key is no longer allowed to push releases?